### PR TITLE
Prepare to release 0.1.3

### PR DESCRIPTION
**Pull request by ccen-stripe**
# Summary

* bump up release version to 0.1.3
* bump up iOS version and update android kotlin version that supports ID/Address
* Update the logic to find the correct viewController for iOS, similar to what payments RN [does](https://github.com/stripe/stripe-react-native/blob/master/ios/StripeSdk.swift#L1184-L1197)

# Motivation

# Testing

- [ ] Added tests
- [ ] Modified tests
- [x] Manually verified

# Screenshots
| Before  | After |
| ------------- | ------------- |
| *before screenshot*  | *after screenshot* |

### CODE REVIEW

**Review by charliecruzan-stripe — APPROVED** [review]

File: android/build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
+    classpath 'com.android.tools.build:gradle:7.4.2'
```
Comment (charliecruzan-stripe) [inline android/build.gradle:13]: i think this means minimum gradle 7.5 is supported right? https://developer.android.com/studio/releases/gradle-plugin

Comment (ccen-stripe) [reply android/build.gradle:13]: yes, is there any concerns on bumping this for payments?

Comment (charliecruzan-stripe) [reply android/build.gradle:13]: I think this means we would stop supporting React Native versions that need below that, so .69 and below: https://github.com/facebook/react-native/blob/0.69-stable/template/android/gradle/wrapper/gradle-wrapper.properties

Comment (charliecruzan-stripe) [reply android/build.gradle:13]: do you need 7.4 specifically? or could we stay at `com.android.tools.build:gradle:7.2.2` for now?

**Pull request merged**